### ISSUE TITLE
gql-gen init - Error: Cannot find module 'graphql/language/parser'

### ISSUE THREAD

**Issue by aoakeson** [dotansimha/graphql-code-generator#1378]
**Describe the bug**
Everytime I run gql-gen I get an error 
```
module.js:549
    throw err;
    ^
Error: Cannot find module 'graphql/language/parser'
```
This is happening in existing and new projects.  I am using Angular CLI to generate my projects. 

**To Reproduce**
Install required packages: 
npm install graphql
npm install -D graphql-code-generator

**Expected behavior**
I expect it to run through the gql init.  It just errors out. 

**Schema/Documents**
If possible and relevant, please provide your GraphQL schema and GraphQL documents.
Not possible.  I cannot run init. 

**Environment:**
 - OS: Windows 10
 - Codegen:
 - Node: 8.11.1

**Additional context**
Here is my package.json
```
{
  "name": "gql-test-project",
  "version": "0.0.0",
  "scripts": {
    "ng": "ng",
    "start": "ng serve",
    "build": "ng build",
    "test": "ng test",
    "lint": "ng lint",
    "e2e": "ng e2e"
  },
  "private": true,
  "dependencies": {
    "@angular/animations": "^6.0.0",
    "@angular/common": "^6.0.0",
    "@angular/compiler": "^6.0.0",
    "@angular/core": "^6.0.0",
    "@angular/forms": "^6.0.0",
    "@angular/http": "^6.0.0",
    "@angular/platform-browser": "^6.0.0",
    "@angular/platform-browser-dynamic": "^6.0.0",
    "@angular/router": "^6.0.0",
    "core-js": "^2.5.4",
    "graphql": "^14.1.1",
    "rxjs": "6.0.0",
    "zone.js": "^0.8.26"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "~0.6.1",
    "@angular/cli": "~6.0.1",
    "@angular/compiler-cli": "^6.0.0",
    "@angular/language-service": "^6.0.0",
    "@types/jasmine": "~2.8.6",
    "@types/jasminewd2": "~2.0.3",
    "@types/node": "~8.9.4",
    "codelyzer": "~4.2.1",
    "graphql-code-generator": "^0.17.0",
    "jasmine-core": "~2.99.1",
    "jasmine-spec-reporter": "~4.2.1",
    "karma": "~1.7.1",
    "karma-chrome-launcher": "~2.2.0",
    "karma-coverage-istanbul-reporter": "~1.4.2",
    "karma-jasmine": "~1.1.1",
    "karma-jasmine-html-reporter": "^0.2.2",
    "protractor": "~5.3.0",
    "ts-node": "~5.0.1",
    "tslint": "~5.9.1",
    "typescript": "~2.7.2"
  }
}

```

**Comment by dotansimha** [comment]
@aoakeson can you try to install `@types/graphql` as well? 
Also, can you share you codegen.yml file? I guess it's an invalid import in one of the plugins.

**Comment by aoakeson** [comment]
Hi @dotansimha I installed they @types/graphql, but still getting the same error.  Like i mentioned, I was trying to run the `gql-gen init` command, but its erroring out.  I also created the codegen.yml file as suggested, and no luck.  Here is that file

```
schema: http://localhost:12465/graphql
generates:
  src/types.ts:
    - typescript-common
    - typescript-apollo-angular

```

**Comment by ardatan** [comment]
You need to have documents to generate Angular services for operations inside documents. I think you don’t have a valid path defined as ‘documents’ in your configuration file.

**Comment by dotansimha** [comment]
Fixed in the recent refactor, we'll release it soon.

**Comment by dotansimha** [comment]
Fix available in 1.0.0 👍
